core/common/src/rpc/web/ITwinCloudRpcRouting.swagger.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
Looking at this again, this is probably all totally wrong because I'm adding support for the new public facing RPC URLs, not REST URLs (of which we are only allowing queryRows and getElementProperties).